{"name": "Принять заказ. Попытка принять заказ без id курьера возвращает корректное сообщение об ошибке", "status": "passed", "description": "тест ручки \"Принять заказ\"", "start": 1724853582227, "stop": 1724853582379, "uuid": "239a7f7a-5126-4a31-aca0-0dc70c8ae278", "historyId": "bd10ff357b9f8b37e0339675f73a8320", "testCaseId": "bd10ff357b9f8b37e0339675f73a8320", "fullName": "tests.TestAcceptOrder#test_try_accept_order_without_courier_id", "labels": [{"name": "suite", "value": "tests"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "FanatovPC"}, {"name": "thread", "value": "31284-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests"}]}